### ISSUE TITLE
Minio Operator v4.1.3 : no matches for kind "Operator" in version "minio.min.io/v2"

### ISSUE THREAD

**Issue by bahetiamit** [minio/operator#726]
## Expected Behavior
Operator should be able to create operator-tls secret.

## Current Behavior
I0714 09:41:42.127242       1 csr.go:170] Certificate successfully fetched, creating secret with Private key and Certificate
E0714 09:41:42.205461       1 operator.go:147] Unexpected error during the creation of the secret/operator-tls: secrets "operator-tls" is forbidden: cannot set blockOwnerDeletion in this case because cannot find RESTMapping for APIVersion minio.min.io/v2 Kind Operator: no matches for kind "Operator" in version "minio.min.io/v2"
I0714 09:41:42.205504       1 main-controller.go:627] Waiting for the operator certificates to be issued secrets "operator-tls" is forbidden: cannot set blockOwnerDeletion in this case because cannot find RESTMapping for APIVersion minio.min.io/v2 Kind Operator: no matches for kind "Operator" in version "minio.min.io/v2"

## Possible Solution
Cause and fix can be seen in https://github.com/minio/operator/issues/520

## Steps to Reproduce (for bugs)

1. init minio 
2. Create tenant
3. Check operator pod logs

**Comment by nitisht** [comment]
closing as duplicate of #720

**Comment by bahetiamit** [comment]
@nitisht I tried with your image - `nitishtiwari/operator:operator-tls-disable` but that seems to be specific to `amd64`. I am using power system (ppc64le). 

Then I built the image locally on power system using commit# 4843d2862a6872c95f3ce5b7a6f405ec9f5547bd  and executing `make build`. 

I then modified the operator deployment to use locally built image and setting up the env variable as suggested.

![image](https://user-images.githubusercontent.com/65336169/126323267-1b8aee0e-77d8-43d9-a58b-abf3d8765fc7.png)


I still get the same error.
